Puppetfile

```diff
@@ -1,15 +1,27 @@
 forge 'https://forgeapi.puppetlabs.com'
 
 # Dependencies
-mod 'puppetlabs/mysql',         '>= 4.0.0'
-mod 'puppetlabs/postgresql',    '>= 4.8.0'
+mod 'katello/candlepin',             :git => 'https://github.com/theforeman/puppet-candlepin'
```

Comment:
Would it be cleaner to further split these inside this file with comments to make obvious katello specific module dependencies?

Reply:
I considered this and had it in an earlier version. On the other hand we have one bunch of modules and don't split those out so not sure what the benefit would be.